<!-- YAML
added: v0.7.3
-->

This event is the same as the one provided by [`child_process.fork()`][].

In a worker you can also use `process.on('error')`.

